ramalama/engine.py

```diff
@@ -333,3 +334,26 @@ def add_labels(args, add_label):
     for arg, label_prefix in label_map.items():
         if value := getattr(args, arg, None):
             add_label(f"{label_prefix}={value}")
+
+
+def wait_for_healthy(args, timeout=20):
```

Comment:
one thing I'm wondering regarding the timeout is if we're sure that podman will never pull the image while this `wait_for_healthy` is counting down

like
```
podman pull image
podman run image
```
and 
```
podman run image
```
these two blocks are semantically similar, but the second `podman run` may take longer before the container comes healthy

Reply:
Made this change:

+        self.ensure_model_exists(args)
         pid = os.fork()